spec/features/admin_claim_approve_spec.rb

```diff
@@ -21,10 +21,7 @@
         find("a[href='#{admin_claim_path(claim_to_approve)}']").click
         perform_enqueued_jobs { click_on "Approve" }
 
-        claim_to_approve.reload
-
-        expect(claim_to_approve.approved_at).to eq(Time.zone.now)
-        expect(claim_to_approve.approved_by).to eq("12345")
+        expect(claim_to_approve.check.checked_by).to eq("12345")
```

Comment:
We should check the creation time, too, since we use that as a check date.

Reply:
Actually I've changed my mind! I don't think we need to do that as it's default Rails behaviour that `created_at` gets set when the record is created. Testing that feels like testing the framework.

Reply:
I'm more thinking about the case where someone changes the behaviour to create the empty check at the start of the claim (or something). We rely on the creation date for the time of check, so we should be confident that its creation date _is_ the time of check.

Reply:
Yeah, I'm not convinced that sort of scenario is very likely to happen. We've defined a `Check` as a RESTfully created model that gets created when someone performs a check. For it to be created at a time other than when the check is performed would be quite odd.

Reply:
Sure.